acceptance/framework/config/config.go

```diff
@@ -59,6 +59,8 @@ type TestConfig struct {
 	VaultServerVersion    string
 
 	NoCleanupOnFailure bool
+	NoCleanupWanFed    bool
```

Comment:
Could you change this from `NoCleanupWanFed` to `NoCleanup`? That way we can use it in other tests to always keep the environment around.